src/main/java/de/retest/recheck/cli/subcommands/Commit.java

```diff
@@ -82,7 +82,10 @@ private void applyChanges( final Persistence<SutState> persistence, final SuiteC
 		try {
 			ApplyChangesToStatesFlow.apply( persistence, suiteChangeSet );
 		} catch ( final NoStateFileFoundException e ) {
-			logger.error( "No state file with name '{}' found!", e.getFilename() );
+			logger.error( "The SUT state file '{}' cannot be found.", e.getFilename() );
```

Comment:
Should we use GM here as well?

Reply:
Yep, I thought this too. But for now I would stick to "SUT state file" because there is a lot of logging comming from recheck that uses this term (see also in `CommitIT`). As soon as we have updated recheck, we can also do it here.